src/plots/plot.ts

```diff
@@ -20,6 +20,11 @@ export module Plots {
   }
 }
 
+interface LightweightPlotEntity extends Plots.PlotEntity {
```

Comment:
The name is a misnomer.  By having the prefix `Lightweight`, this implies that there are less items in `LightweightPlotEntity` than `Plots.PlotEntity` which is not the case.